docs/source/getting-started.rst

```diff
@@ -1,48 +1,246 @@
+      --protocol email --notification-endpoint EMAIL@DOMAIN.COM
+
+.. note:: You will need to click the verification link in your email to activate the subscription.
+
+3. Add the ``streamalert-test-data`` SNS topic as an input to the (default) ``prod`` `cluster <clusters.html>`_.
+Open ``conf/clusters/prod.json`` and change the ``stream_alert`` section to look like this:
+
+.. code-block:: json
+
+  {
+    "stream_alert": {
+      "rule_processor": {
+        "enable_metrics": true,
+        "inputs": {
+          "aws-sns": [
+            "arn:aws:sns:REGION:ACCOUNTID:streamalert-test-data"
+          ]
+        },
+        "log_level": "info",
+        "memory": 128,
+        "timeout": 10
+      }
+    }
+  }
+
+4. Tell StreamAlert which `log schemas <conf-schemas.html>`_ will be sent to this input.
+Open ``conf/sources.json`` and change the ``sns`` section to look like this:
+
+.. code-block:: json
+
+  {
+    "sns": {
+      "streamalert-test-data": {
+        "logs": [
+          "cloudwatch"
+        ]
+      }
+    }
+  }
+
+5. Add the alert topic as a StreamAlert output:
+
+.. code-block:: bash
+
+  $ ./manage.py output new --service aws-sns
+
+  Please supply a short and unique descriptor for this SNS topic: test-email
+
+  Please supply SNS topic name: streamalert-test-alerts
+
+If you look at ``conf/outputs.json``, you'll notice that the SNS topic was automatically added.
+
+6. Configure a rule to send to the alerts topic.
+We will use ``rules/community/cloudtrail/cloudtrail_root_account_usage.py`` as an example, which
+alerts on any usage of the root AWS account. Change the rule decorator to:
+
+.. code-block:: python
+
+  @rule(
+      logs=['cloudwatch:events'],
+      req_subkeys={'detail': ['userIdentity', 'eventType']},
+      outputs=['aws-sns:test-email']  # Add this line
+  )
+  def cloudtrail_root_account_usage(rec):
+
+7. Now we need to update StreamAlert with these changes:
+
+.. code-block:: bash
+
+  # Hook the streamalert-test-data SNS topic up to the StreamAlert rule processor
+  ./manage.py terraform build
+
+  # Deploy a new version of all of the Lambda functions with the updated rule and config files
+  ./manage.py lambda deploy -p all
+
+8. Time to test! Create a file named ``cloudtrail-root.json`` with the following contents:
+
+.. code-block:: json
+
+  {
+    "account": "1234",
+    "detail": {
+      "eventType": "AwsConsoleSignIn",
+      "userIdentity": {
+        "type": "Root"
+      }
+    },
+    "detail-type": "CloudTrail Test",
+    "id": "1234",
+    "region": "us-east-1",
+    "resources": [],
+    "source": "1.1.1.2",
+    "time": "now",
+    "version": "2018"
+  }
+
+This is only a rough approximation of what the real log might look like, but good enough for our purposes.
+Then send it off to the data SNS topic:
+
+.. code-block:: bash
+
+  aws sns publish --topic-arn arn:aws:sns:REGION:ACCOUNTID:streamalert-test-data --message "`cat cloudtrail-root.json`"
```

Comment:
if you didn't want to use back ticks here, you could use command expansion with `$` like:

`... --message "$(cat cloudtrail-root.json)"`

not necessary, just a suggestion :)